api/client/proto/authservice.proto

```diff
@@ -1132,4 +1138,6 @@ service AuthService {
     rpc GetEvents(GetEventsRequest) returns (Events);
     // In-session request for audit events.
     rpc GetSessionEvents(GetSessionEventsRequest) returns (Events);
+    // StreamSessionEvents streams audit events from a given session recording.
+    rpc StreamSessionEvents(StreamSessionEventsRequest) returns (stream events.OneOf);
```

Comment:
Would it be possible to resume the stream from some event (e.g. if download was interrupted) to make it symmetrical with the other stream or we will omit this from MVP?

Reply:
Hmm, looks possible. I'll have a go at it.

Reply:
@klizhentas This seems a tad tricky to do since print events lack UIDs.

Reply:
yes, because there so many print events, having UID for each one would have wasted a lot of memory. However, every event in the stream is sequentially numbered, so it should be possible to resume the download of the stream from the last sequence, you can see example of how resume works from the upload side here:

https://github.com/gravitational/teleport/blob/4d36870ff0334c8f1a5dc75d5f8208a43aa9e47f/lib/events/auditwriter.go#L483

The client keeps the last event index that was uploaded, so when upload interrupts it replays the events from the last confirmed index

Reply:
Added a parameter to resume at a specific event index.